### DISCUSSION TITLE
How to manually modify the status of a field?

### DISCUSSION THREAD

**Discussion by blackawn** [TanStack/form#921] [Q&A]
Can I manually modify the status of the field?

For example, when I register, the form content is successfully verified, and the user name already exists when the request is returned.

At this time, I want to give this username field a new `error` prompt

**Comment by richin13** [comment]
Out of curiosity, How did you resolve this?

**Reply by blackawn** [reply]
> Out of curiosity, How did you resolve this?

```javascript
form.setFieldMeta('account', (meta) => ({
  ...meta,
  errorMap: {
    onSubmit: `Username ${value.account} is taken`,
  },
}));
```
